utils/remote.c

```diff
@@ -645,6 +647,28 @@ static void addDays(tr_variant* args, tr_quark const key, char const* arg)
     }
 }
 
+static void addLabels(tr_variant* args, const tr_quark key, const char* arg)
+{
+    tr_variant* labels = tr_variantDictAddList(args, key, 100);
+    while (arg && *arg)
+    {
+        const char* pch = strchr(arg, ',');
+        char* str;
+        if (pch)
+        {
+            str = tr_strndup(arg, pch - arg);
+            arg = pch + 1;
+        }
+        else
+        {
+            str = tr_strdup(arg);
+            arg += strlen(arg);
+        }
+        tr_variantListAddStr(labels, str);
+        tr_free(str);
+    }
+}
```

Comment:
This pattern is showing up in a couple of places, e.g. https://github.com/transmission/transmission/pull/229/files#diff-9905fe306ee1f93d2363fe0a9412df3dR2388

Maybe it would be useful to extract this logic into a reusable util function to split a string into tokens?